src/borg/archiver.py

```diff
@@ -534,6 +534,7 @@ def create_inner(archive, cache, fso):
                     process_file_chunks=cp.process_file_chunks, add_item=archive.add_item,
                     chunker_params=args.chunker_params, show_progress=args.progress)
                 create_inner(archive, cache, fso)
+                logger.error("Repository %s has been created successfully" % archive.name)
```

Comment:
Why output it on error level? Check the available logging levels to find something more appropriate.

Also, you are saying "Repository %s", but what you fill into the placeholder is the archive's name, not the repository URL.